docker/dmd.sh

```diff
@@ -4,17 +4,9 @@
 #   - dmd-bin package version (default: latest)
 set -xe
 
-dmd_version=""
-test -n "$1" &&
-        dmd_version="=$1"
+dmd_version="2.071.2"
 
-# Install D-APT packages
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EBCF975E5BA24D5E && \
-        wget http://master.dl.sourceforge.net/project/d-apt/files/d-apt.list \
-            -O /etc/apt/sources.list.d/d-apt.list
+apt -y install xdg-utils libcurl3
 
-# Make sure our packages list is updated
-apt-get update
-
-# Install dmd-bin
-apt-get install -y dmd-bin$dmd_version
+wget -O dmd.deb http://downloads.dlang.org/releases/2.x/${dmd_version}/dmd_${dmd_version}-0_amd64.deb
```

Comment:
Does installing specific version work on your local system? I had similar errors to ones in travis when tried myself.